nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -95,6 +95,10 @@ let
       # Tell systemd about our $DISPLAY. This is needed by the
       # ssh-agent unit.
       ${config.systemd.package}/bin/systemctl --user import-environment DISPLAY
+      ${config.systemd.package}/bin/systemctl --user import-environment XAUTHORITY
+
+      # Tell systemd about the dbus session bus address. This is required by user units using the session bus.
+      ${config.systemd.package}/bin/systemctl --user import-environment DBUS_SESSION_BUS_ADDRESS
```

Comment:
The following should be possible according to man page:

```nix
${config.systemd.package}/bin/systemctl --user import-environment DISPLAY XAUTHORITY DBUS_SESSION_BUS_ADDRESS
```

Reply:
Yes it's possible but I wanted it on multiple lines for better clarity and commenting why every variable is necessary.

Reply:
less commands -> faster